dsgrid/api/app.py

```diff
@@ -151,66 +153,64 @@ async def list_project_dimensions(project_id: str):
 
 
 @app.get(
-    "/projects/{project_id}/dimensions/dimension_query_names",
-    response_model=GetProjectDimensionQueryNamesResponse,
+    "/projects/{project_id}/dimensions/dimension_names",
+    response_model=GetProjectDimensionNamesResponse,
 )
-async def get_project_dimension_query_names(project_id: str):
+async def get_project_dimension_names(project_id: str):
     """List the base and supplemental dimension query names for the project by type."""
     mgr = manager.project_manager
     project = mgr.get_by_id(project_id)
-    return GetProjectDimensionQueryNamesResponse(
+    return GetProjectDimensionNamesResponse(
         project_id=project_id,
-        dimension_query_names=project.get_dimension_query_names_model(),
+        dimension_names=project.get_dimension_names_model(),
     )
 
 
 @app.get(
-    "/projects/{project_id}/dimensions/base_dimension_query_name/{dimension_type}",
-    response_model=GetProjectBaseDimensionQueryNameResponse,
+    "/projects/{project_id}/dimensions/base_dimension_name/{dimension_type}",
+    response_model=GetProjectBaseDimensionNameResponse,
 )
-async def get_project_base_dimension_query_name(project_id: str, dimension_type: DimensionType):
+async def get_project_base_dimension_name(project_id: str, dimension_type: DimensionType):
     """Get the project's base dimension query name for the given dimension type."""
     mgr = manager.project_manager
     config = mgr.get_by_id(project_id)
-    return GetProjectBaseDimensionQueryNameResponse(
+    return GetProjectBaseDimensionNameResponse(
         project_id=project_id,
         dimension_type=dimension_type,
-        dimension_query_name=config.get_base_dimension(dimension_type).model.dimension_query_name,
+        dimension_name=config.get_base_dimension(dimension_type).model.name,
     )
 
 
 @app.get(
-    "/projects/{project_id}/dimensions/supplemental_dimension_query_names/{dimension_type}",
-    response_model=ListProjectSupplementalDimensionQueryNames,
+    "/projects/{project_id}/dimensions/supplemental_dimension_names/{dimension_type}",
```

Comment:
Do you need the f str for these {} inserts?

Reply:
They aren't f-strings. Those are placeholders for path parameters. The user will enter something like `http://localhost:3000/projects/my-project-id/dimensions/supplemental_dimension_names/geography`